src/metabase/automagic_dashboards/core.clj

```diff
@@ -206,18 +211,21 @@
   (comp qp.util/query->source-card-id :dataset_query))
 
 (def ^:private ^{:arglists '([card-or-question])} native-query?
-  (comp #{:native} qp.util/normalize-token #(qp.util/get-in-normalized % [:dataset_query :type])))
+  (comp #{:native} qp.util/normalize-token #(get-in % [:dataset_query :type])))
 
 (def ^:private ^{:arglists '([card-or-question])} source-question
   (comp Card qp.util/query->source-card-id :dataset_query))
 
 (defn- table-like?
   [card-or-question]
-  (let [[aggregation & _] (qp.util/get-in-normalized card-or-question [:dataset_query :query :aggregation])]
-    (or (nil? aggregation)
-        (and (or (string? aggregation)
-                 (keyword? aggregation))
-             (= (qp.util/normalize-token aggregation) :rows)))))
+  (nil? (get-in card-or-question [:dataset_query :query :aggregation])))
+
+(defn- table-id
+  "For some reason that's not really clear things like the 'Cards' used in this namespace don't have consistent Table ID
+  keys..."
```

Comment:
This was to support both Cards loaded from the DB as well as those that came from a roundtrip to the FE. If my understanding is correct, we now don't munge keys when sending to the FE, so I think it should be sufficient to just use `table-id` everywhere.

Reply:
The thing is cards from the DB still have a `table_id` key. I'm not normalizing things like that (although we certainly could normalize all the DB keys; that's a different story). I'll update the docstring

Reply:
I think it might be the things you call "Questions" (?) have `table-id`. Things coming back from a round trip from the FE should still have `table_id`

Reply:
With "question" I mean an adhoc query. In terms of x-ray code these come back from the FE and get stuffed into a `QueryInstance` via models/query/adhoc-query

Reply:
ok got it.